docker/build-tools/Dockerfile

```diff
@@ -118,25 +118,10 @@ ADD https://github.com/cli/cli/releases/download/v${GH_VERSION}/gh_${GH_VERSION}
 RUN dpkg -i /tmp/gh_${GH_VERSION}_linux_${TARGETARCH}.deb
 RUN mv /usr/bin/gh ${OUTDIR}/usr/bin
 
-# Install protoc-gen-validate
```

Comment:
I think we can drop this entirely actually

Reply:
Removed. I see you were doing some work to remove this anyway. Not sure if there's anything we need to do in Envoy. I guess we will see.

Reply:
Envoy pulls it from bazel not here anyways, and we don't use it in our own Istio protos. The removal in Envoy is technically orthogonal from removal here actually